packages/storefront-lwc/src/modules/commerce/productSearchResults/productSearchResults.js

```diff
@@ -5,45 +5,66 @@
     For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
 */
 import { LightningElement, wire, track, api } from 'lwc';
-import { productsByQuery } from 'commerce/data';
+import QUERY from './gqlTemplate';
```

Comment:
The gqlTemplate file name seems very generic should we consider renaming the file? If this is a pattern we want to follow we should also move the QUERY in product details into a gqlTemplate inside the productDetail directory

Reply:
This was moved out b/c of the file size of productSearchResults was too big, can ask the team if we should make this a general pattern

Reply:
@abkap02  is working on pdp now for add to basket support and externalizing the gql. So maybe he can make that part of his task.

Reply:
@coopaq: roger that

Reply:
already added it in

Reply:
also changed the name of the file to gqlQuery.js